### DISCUSSION TITLE
About document translation

### DISCUSSION THREAD

**Discussion by Alfred-Skyblue** [unocss/unocss#2474] [General]
I found that the documentation of unocss is being improved, I can translate it into Chinese, is it acceptable to the official?

**Comment by zyyv** [comment]
https://github.com/unocss/unocss/pull/2439#issuecomment-1495941370

**Reply by Alfred-Skyblue** [reply]
> [#2439 (comment)](https://github.com/unocss/unocss/pull/2439#issuecomment-1495941370)

Thank you for your reply. I have translated the official document. I hope my translation can help more people use unocss. I will continue to update it. If you have any questions, please feel free to contact me, thank you!

[Document](https://alfred-skyblue.github.io/unocss-docs-cn/)

**Comment by Alfred-Skyblue** [comment]
Can it be linked to the official docs? 

[GitHub](https://github.com/Alfred-Skyblue/unocss-docs-cn)
[Document](https://alfred-skyblue.github.io/unocss-docs-cn/)
